airbyte-webapp/src/components/connection/CatalogTree/next/CatalogTreeTableHeader.tsx

```diff
@@ -85,9 +85,15 @@ export const CatalogTreeTableHeader: React.FC = () => {
       </TextCell> */}
       <HeaderCell>
         <FormattedMessage id="form.namespace" />
+        <InfoTooltip>
+          <FormattedMessage id="connectionForm.destinationName.info" />
+        </InfoTooltip>
       </HeaderCell>
       <HeaderCell>
         <FormattedMessage id="form.streamName" />
+        <InfoTooltip>
+          <FormattedMessage id="connectionForm.destinationStream.info" />
```

Comment:
We need to create new strings for sourceName and souceStream because these talk about the destination, but these are about the source.

Reply:
Fixed with temporary tooltip strings